python/kserve/kserve/utils/utils.py

```diff
@@ -133,3 +134,31 @@ def to_headers(context: ServicerContext) -> Dict[str, str]:
         headers[metadatum.key] = metadatum.value
 
     return headers
+
+
+def get_predict_input(payload: Union[Dict, InferRequest]):
```

Comment:
Add return type which is`Union[Dict, np.array]`

Reply:
@yuzisun return type is `Union[List, np.ndarray]` correct?

Reply:
you are right